### PR TITLE
roe-1986: Feature flag page content

### PR DESCRIPTION
**Pull request by JoshFen95**
- add feature flag check around interrupt page content
- if trust flag is false then information regarding the trust spreadsheet is displayed on the interrupt page
- add test to ensure that trust excel information is contained on the page when trust feature flag is false.

### JIRA link

[ROE-1986](https://companieshouse.atlassian.net/browse/ROE-1986)

### Work checklist

- [X] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

[ROE-1986]: https://companieshouse.atlassian.net/browse/ROE-1986?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: test/controllers/interrupt.card.controller.spec.ts [hunk 1]
```diff
@@ -12,13 +13,15 @@ import { INTERRUPT_CARD_PAGE_TITLE } from "../__mocks__/text.mock";
...
 import { isSecureRegister } from "../../src/middleware/navigation/is.secure.register.middleware";
+import { isActiveFeature } from "../../src/utils/feature.flag";
 
 const mockIsSecureRegisterMiddleware = isSecureRegister as jest.Mock;
 mockIsSecureRegisterMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
 
 const mockAuthenticationMiddleware = authentication as jest.Mock;
 mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
 
+
```
Comment (okoye-companieshouse) [inline test/controllers/interrupt.card.controller.spec.ts:24]: Can this extra line be removed?

File: views/interrupt-card.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
   {% include "includes/back-link.html" %}
 {% endblock %}
 
+
```
Comment (okoye-companieshouse) [inline views/interrupt-card.html:11]: Another extra line

**Review by okoye-companieshouse — APPROVED** [review]

File: test/controllers/interrupt.card.controller.spec.ts (hunk 1 above)
Comment (JoshFen95) [reply test/controllers/interrupt.card.controller.spec.ts]: good spot, removed

File: views/interrupt-card.html (hunk 2 above)
Comment (JoshFen95) [reply views/interrupt-card.html]: good spot, removed

**Pull request merged**